src/main/java/dev/jbang/util/Util.java

```diff
@@ -427,22 +444,44 @@ public static Path downloadFileSwizzled(String fileURL, File saveDir) throws IOE
 	static private String getAgentString() {
 		if (agent == null) {
 			String version = System.getProperty("java.version");
-			agent = "JBang/" + BuildConfig.VERSION + " (" + System.getProperty("os.name") + "/"
+			agent = "JBang/" + getJbangVersion() + " (" + System.getProperty("os.name") + "/"
 					+ System.getProperty("os.version") + "/" + System.getProperty("os.arch") + ") " + "Java/" + version;
```

Comment:
Should add Java.vendor too

Reply:
WAnt me to put that in this PR?

Reply:
Feel free. I can also add it before releasing.

Reply:
I'll leave it to you then because you have a better idea of what it is you want exactly :-)